charts/latest-v2/azuredisk-csi-driver/templates/csi-azuredisk-node-windows-hostprocess.yaml

```diff
@@ -154,22 +183,9 @@ spec:
           command:
             - "azurediskplugin.exe"
           args:
-            - "--kubeconfig=.\\var\\run\\secrets\\kubernetes.io\\serviceaccount\\kubeconfig.conf"
             - "--v={{ .Values.node.logLevel }}"
-            - "--endpoint=$(CSI_ENDPOINT)"
             - "--nodeid=$(KUBE_NODE_NAME)"
-            - "--metrics-address=0.0.0.0:{{ .Values.node.metricsPort }}"
-            - "--is-node-plugin=true"
-            - "--drivername={{ .Values.driver.name }}"
-            - "--driver-object-namespace={{ .Values.driver.objectNamespace }}"
-            - "--volume-attach-limit={{ .Values.driver.volumeAttachLimit }}"
-            - "--cloud-config-secret-name={{ .Values.node.cloudConfigSecretName }}"
-            - "--cloud-config-secret-namespace={{ .Values.node.cloudConfigSecretNamespace }}"
-            - "--custom-user-agent={{ .Values.driver.customUserAgent }}"
-            - "--user-agent-suffix={{ .Values.driver.userAgentSuffix }}"
-            - "--allow-empty-cloud-config={{ .Values.node.allowEmptyCloudConfig }}"
-            - "--support-zone={{ .Values.node.supportZone }}"
-            - "--get-node-info-from-labels={{ .Values.windows.getNodeInfoFromLabels }}"
+            - "--config=/etc/csi-azuredisk-node-win-hp/config.yaml"
```

Comment:
I believe this should be `"--config=.\\etc\\csi-azuredisk-node-win-hp\\config.yaml"`.  Paths through Volume Mounts are accessed in Host Process Containers using a relative path from the current working directory or abosolute path using `$CONTAINER_SANDBOX_MOUNT_POINT` as the root. See [Create Windows HostProcess Pod / Volume Mount](https://kubernetes.io/docs/tasks/configure-pod-container/create-hostprocess-pod/#volume-mounts) for details.

Reply:
Should it also be applied to csi-azuredisk-node-windows.yaml?

Reply:
yes